src/Files.App/Data/Commands/Manager/CommandManager.cs

```diff
@@ -199,149 +199,149 @@ public CommandManager()
+			[CommandCodes.NewTab] = new DebouncedActionDecorator(new NewTabAction()),
+			[CommandCodes.FormatDrive] = new DebouncedActionDecorator(new FormatDriveAction()),
+			[CommandCodes.NavigateBack] = new DebouncedActionDecorator(new NavigateBackAction()), 
+			[CommandCodes.NavigateForward] = new DebouncedActionDecorator(new NavigateForwardAction()),
+			[CommandCodes.NavigateUp] = new DebouncedActionDecorator(new NavigateUpAction()),
+			[CommandCodes.DuplicateCurrentTab] = new DebouncedActionDecorator(new DuplicateCurrentTabAction()),
+			[CommandCodes.DuplicateSelectedTab] = new DebouncedActionDecorator(new DuplicateSelectedTabAction()),
+			[CommandCodes.CloseTabsToTheLeftCurrent] = new DebouncedActionDecorator(new CloseTabsToTheLeftCurrentAction()),
+			[CommandCodes.CloseTabsToTheLeftSelected] = new DebouncedActionDecorator(new CloseTabsToTheLeftSelectedAction()),
+			[CommandCodes.CloseTabsToTheRightCurrent] = new DebouncedActionDecorator(new CloseTabsToTheRightCurrentAction()),
+			[CommandCodes.CloseTabsToTheRightSelected] = new DebouncedActionDecorator(new CloseTabsToTheRightSelectedAction()),
+			[CommandCodes.CloseOtherTabsCurrent] = new DebouncedActionDecorator(new CloseOtherTabsCurrentAction()),
+			[CommandCodes.CloseOtherTabsSelected] = new DebouncedActionDecorator(new CloseOtherTabsSelectedAction()),
+			[CommandCodes.OpenDirectoryInNewPane] = new DebouncedActionDecorator(new OpenDirectoryInNewPaneAction()),
+			[CommandCodes.OpenDirectoryInNewTab] = new DebouncedActionDecorator(new OpenDirectoryInNewTabAction()),
+			[CommandCodes.OpenInNewWindowItem] = new DebouncedActionDecorator(new OpenInNewWindowItemAction()),
+			[CommandCodes.ReopenClosedTab] = new DebouncedActionDecorator(new ReopenClosedTabAction()),
+			[CommandCodes.PreviousTab] = new DebouncedActionDecorator(new PreviousTabAction()),
+			[CommandCodes.NextTab] = new DebouncedActionDecorator(new NextTabAction()),
+			[CommandCodes.CloseSelectedTab] = new DebouncedActionDecorator(new CloseSelectedTabAction()),
+			[CommandCodes.OpenNewPane] = new DebouncedActionDecorator(new OpenNewPaneAction()),
+			[CommandCodes.ClosePane] = new DebouncedActionDecorator(new ClosePaneAction()),
+			[CommandCodes.OpenFileLocation] = new DebouncedActionDecorator(new OpenFileLocationAction()),
+			[CommandCodes.PlayAll] = new DebouncedActionDecorator(new PlayAllAction()),
+			[CommandCodes.GitFetch] = new DebouncedActionDecorator(new GitFetchAction()),
+			[CommandCodes.GitInit] = new DebouncedActionDecorator(new GitInitAction()),
+			[CommandCodes.GitPull] = new DebouncedActionDecorator(new GitPullAction()),
+			[CommandCodes.GitPush] = new DebouncedActionDecorator(new GitPushAction()),
+			[CommandCodes.GitSync] = new DebouncedActionDecorator(new GitSyncAction()),
+			[CommandCodes.OpenAllTaggedItems] = new DebouncedActionDecorator(new OpenAllTaggedActions(), TimeSpan.FromMilliseconds(1000))
```

Comment:
Not sure that adding an example is useful. After all, if someone needs one day to add a new command, their IDE should display the constructor with the possibility of changing the timespan. Here is just adds a "why is this command different from others?" moment when reading it.